src/quickgui/qgsquickcoordinatetransformer.h

```diff
@@ -48,7 +48,7 @@ class QUICK_EXPORT QgsQuickCoordinateTransformer : public QObject
     Q_PROPERTY( QgsCoordinateReferenceSystem sourceCrs READ sourceCrs WRITE setSourceCrs NOTIFY sourceCrsChanged )
 
     //! Map settings, for getting transformation context
-    Q_PROPERTY( QgsQuickMapSettings *mapSettings MEMBER mMapSettings NOTIFY mapSettingsChanged )
+    Q_PROPERTY( QgsQuickMapSettings *mapSettings READ mapSettings WRITE setMapSettings NOTIFY mapSettingsChanged )
```

Comment:
hmm, is the new implementation working? if it is needed just for transformation context, then in the setMapSettings() you need to connect some signals to check if context is changed? no?

if there is no such signal, maybe we can have just a context here as a property and not whole mapSettings?